text/chapter10.md

```diff
@@ -1073,7 +1071,7 @@ Left errs -> alert $ "There are " <> show (length errs) <> " validation errors."
 alert $ "Error: " <> err <> ". Loading examplePerson"
 ```
 
- ## Exercises
+## Exercises
```

Comment:
This indentation may be a workaround to get mdbook rendering correctly

Reply:
I' m not sure about this one. This is because the mdBook conversion result seems to be the same whether the indentation is present or not.